### ISSUE TITLE
Banner label link contrast failure on focus

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/lively#40]
With focus, linked labels within configured banners have only a 3.6:1 contrast, under the 4.5:1 minimum. The text becomes `rgb(51, 0, 78)` on the `rgb(214, 45, 106)` background.

**Comment by nelsonamaya82** [comment]
Thanks @aviannamiller, this is fixed and merged into the `master` branch.

**Comment by aviannamiller** [comment]
Fantastic, thank you.
